--- luasrc/controller/quickstart.lua	2025-09-20 11:50:58
+++ luasrc/controller/quickstart.lua	2025-09-20 11:46:46
@@ -15,13 +15,8 @@
         if nixio.fs.access("/usr/lib/lua/luci/view/quickstart/main_dev.htm") then
             entry({"admin", "quickstart_dev"}, call("quickstart_dev", {index={"admin", "quickstart_dev"}})).leaf = true
         end
-        entry({"admin", "nas", "raid"}, call("quickstart_index", {index={"admin", "nas"}}), _("RAID"), 10).leaf = true
-        entry({"admin", "nas", "smart"}, call("quickstart_index", {index={"admin", "nas"}}), _("S.M.A.R.T."), 11).leaf = true
         entry({"admin", "network", "interfaceconfig"}, call("quickstart_index", {index={"admin", "network"}}), _("NetworkPort"), 11).leaf = true

-        entry({"admin", "nas", "quickstart"}).dependent = false
-        entry({"admin", "nas", "quickstart", "auto_setup"}, post("auto_setup"))
-        entry({"admin", "nas", "quickstart", "setup_result"}, call("setup_result"))
     else
         entry({"admin", "quickstart"}, call("redirect_fallback")).leaf = true
     end
